server/selinux/server/relabel.sh

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env bash
 
+function version_less_than {
+# Determines if the version passed in as the first argument is less than the version in the second
+# argument.
+    [[ $(echo -e $1'\n'$2|sort -V|head -n 1) != $2 ]]
```

Comment:
I did several comparisons locally, and I think this statement is exactly what we need.